app/views/trips/card/_trip_card.html.erb

```diff
@@ -38,7 +38,19 @@
             
           <% end %>
         </div>
-
+        <% @duties_done = trip.checklist.duties.count { |duty| duty.status == "✅" } %>
+        <% @duties_all = trip.checklist.duties.count %>
+        <% if trip.checklist_items.count { |item| item.duties.first != nil } == trip.checklist_items.count %>
+          <div class="progress">
+            <div class="progress-bar" role="progressbar" style="width: <%= (@duties_done/@duties_all.to_f*100).round %>%" aria-valuenow="<%= (@duties_done/@duties_all.to_f*100).round %>" aria-valuemin="0" aria-valuemax="100">
```

Comment:
Line is breaking the app for new users
<img width="1676" alt="Screen Shot 2021-04-16 at 8 15 31 PM" src="https://user-images.githubusercontent.com/10403497/115095862-cf2b6780-9ef0-11eb-9663-30197bca3c44.png">